scripts/keymigrate/migrate.go

```diff
@@ -26,9 +26,12 @@ type (
 	migrateFunc func(keyID) (keyID, error)
 )
 
-func getAllLegacyKeys(db dbm.DB) ([]keyID, error) {
+func getAllLegacyKeys(db dbm.DB, storeName string) ([]keyID, error) {
 	var out []keyID
 
+	// TODO: there is a theoretical optimization where we build a
```

Comment:
I think this would be tricky to get right, since there are multiple candidate ranges (though possibly not bracketing? Anyway not likely to save that much I believe)